eng/GenerateAnalyzerNuspec.targets

```diff
@@ -119,7 +119,7 @@
   
   <PropertyGroup>
     <!-- Ideally, we would extract this from the MSBuild task, but we need this as the Target Output before that task is executed -->
-    <_GenerateAnalyzerNuspecPath>$(ArtifactsBinDir)GenerateAnalyzerNuspec\$(Configuration)\net7.0\GenerateAnalyzerNuspec.dll</_GenerateAnalyzerNuspecPath>
+    <_GenerateAnalyzerNuspecPath>$(ArtifactsBinDir)GenerateAnalyzerNuspec\$(Configuration)\$(NetCurrent)\GenerateAnalyzerNuspec.dll</_GenerateAnalyzerNuspecPath>
```

Comment:
Where is this property $(NetCurrent) set?

Reply:
@mavasani Introduced by Arcade in https://github.com/dotnet/arcade/pull/12161

It was advised that we use that https://github.com/dotnet/roslyn-analyzers/issues/6490